tests/Resources/DataFixtures/Page/template_with_xinclude_properties.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" ?>
+<properties xmlns="http://schemas.sulu.io/template/template"
```

Comment:
the testcase should only test this little usecase wether the include works or not. everything else should be tested othwise. it is better readable if you dont have a huge array in the test-class